File: k01-logica-de-programacao/cap-05/controle-de-fluxo/ADivisivelPorB.cs
/* VERSAO 1 */
/*
class ADivisivelPorB
{
	static void Main ()
	{

		int a = 246;
	
		int b = 3;
	
		if (a % b == 0)
		{

			System.Console.WriteLine("É divisível");
		}
		else
		{
			
			System.Console.WriteLine("Não é divisível");
		}
	}
}
*/

/* VERSAO 2 */
/*
class ADivisivelPorB
{
	static void Main ()
	{
		System.Random gerador = new System.Random();

		int a = gerador.Next (1000);

		int b = gerador.Next (5) + 1;

		System.Console.WriteLine("a = " + a);

		System.Console.WriteLine("b = " + b);

		if ( a % b == 0)
		{
			
			System.Console.WriteLine("É divisível");
		}
		else
		{
			
			System.Console.WriteLine("Não é divisível");
		}
	}
}
*/

File: k01-logica-de-programacao/cap-05/controle-de-fluxo/ADivisivelPorB.java
/* VERSAO 1 */
/*
class ADivisivelPorB {
	public static void main(String[] args) {
		
		int a = 246;

		int b = 3;
		
		if (a % b == 0) {

		System.out.println("É divisível");
		
		} else {

		System.out.println("Não é divisível");
		}
	}
}
*/

/* VERSAO 2 */
/*
class ADivisivelPorB {
	public static void main(String[] args) {
		java.util.Random gerador = new java.util.Random();
		
		int a = gerador.nextInt (1000);
		
		int b = gerador.nextInt (5) + 1;
		
		System.out.println("a = " +a);
		
		System.out.println("b = " + b);
			
		if (a % b == 0) {
			
			System.out.println("É divisível ");
		
		} else {

			System.out.println("Não é divisível");
		}
	}
}
*/